src/sources/file.rs

```diff
@@ -953,16 +953,20 @@ mod tests {
         }
     }
 
+    #[cfg(target_os = "linux")] // see #7988
+    #[ignore]
```

Comment:
Um, did you mean to leave in the ignore here?

Reply:
I sure didn't. I meant to push an amended commit before I went to lunch. Corrected with the unpushed commit.